Library/Homebrew/test/rubocops/lines/generate_completions_spec.rb

```diff
@@ -0,0 +1,78 @@
+# typed: false
+# frozen_string_literal: true
+
+require "rubocops/lines"
+
+describe RuboCop::Cop::FormulaAudit do
```

Comment:
I don't think this top-level `describe` is needed

Reply:
Without: 

```
Library/Homebrew/test/rubocops/lines/generate_completions_spec.rb:6:1: C: Do not use multiple top-level example groups - try to nest them.
describe RuboCop::Cop::FormulaAudit::GenerateCompletionsDSL do
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```